finetune/saver.py

```diff
@@ -35,8 +35,8 @@ def save(self, finetune_obj, path, mkdir=True):
             fallback = joblib.load(self.fallback_filename)
         included, excluded = self.find_trainable_variables()
 
-        if not all(var.name in fallback for var in excluded):
-            warnings.warn("Attempting to do a partial save where variables are excluded that do not have a "
+        if not all((var.name in fallback) or (var not in tf.trainable_variables()) for var in excluded):
```

Comment:
Double negative is gross.  Why not `if any((var.name not in fallback and var in tf.trainable_variables()) for var in excluded)`?